### PR TITLE
Allow approximately equal signs to work on tempo text

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #19819

### CODE REVIEW

**Comment by worldwideweary** [comment]
Seems okay to me `>;]` (figure its better than the freakin' 😁 emoji being used these days)

 I'm not sure about what you mentioned regarding _circa/approximate_ . So long as the regex identifier finds "=", having some extra words shouldn't be problematic. Who knows though?

**Comment by worldwideweary** [comment]
Wouldn't writing, for example: `  ♩= 125 ca.` confuse the regex parser if `ca.` were to be equivocated to an equals sign?

**Comment by Jojo-Schmitz** [comment]
when in doubt, try it out ;-)

**Comment by Jojo-Schmitz** [comment]
I just did: it works

**Comment by worldwideweary** [comment]
> when in doubt, try it out ;-)
> 
> I just did: it works.
> 
> And `approx. ♩ = 188 ` works too

Sounds good to me

File: src/engraving/dom/tempotext.cpp [hunk 1]
```diff
@@ -256,6 +256,11 @@ void TempoText::updateTempo()
     String s = plainText();
     s.replace(u",", u".");
     s.replace(u"<sym>space</sym>", u" ");
+    s.replace(u"≒", u"=");
+    s.replace(u"≈", u"=");
+    s.replace(u"~", u"=");
+    s.replace(u"ca.", u"=");
+    s.replace(u"approx.", u"=");
```
Comment (cbjeukendrup) [inline src/engraving/dom/tempotext.cpp:263]: Wouldn't it make more sense to replace these with `""`?

For example, if you have `\uECA5 = ca. 110`, and replace ca. with =, you get `\uECA5 = = 110` which doesn't match the regex.

Comment (Jojo-Schmitz) [reply src/engraving/dom/tempotext.cpp:263]: Unless it is  `♩ ca. 188`. But yes, that seems unlikely, I'll chane as per your proposal

**Review by cbjeukendrup — APPROVED** [review]

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved
https://github.com/musescore/MuseScore/issues/19819 FIXED

**Pull request merged**